Blog/blogForms.py

```diff
@@ -13,12 +14,13 @@ class Meta:
         fields = ['title', 'content']
         widgets = {
             'title': TextInput(),
-            'content': Textarea(attrs={'rows': 15}),
+            'content': MarkdownxFormField(),
         }
 
         labels = {
             'title': _('Title'),
-            'content': _('Content')
+            # intentionally set empty
```

Comment:
Please add the reason why `content` is set to an empty string. What does this do or not do?